### ISSUE TITLE
Personally, I find 8 spaces for each tab really a lot ;) 

### ISSUE THREAD

**Issue by godspeedelbow** [faradji/Server-Side-REST-API#5]
2 maybe? 

https://github.com/faradji/Server-Side-REST-API/blob/87b9279b8329080d5a613e079ebd62e6f50b7a66/config.js#L4

**Comment by faradji** [comment]
Yeah, in Atom I have two tabs only, not sure why it got to 8 in github hehe
